handbook/policies_and_training.md

```diff
@@ -0,0 +1,349 @@
+2. **We want employees to be able to think about the best interests of the company, even if that means their own employment relationship with the company would end.** It is a perhaps-counterintuitive outcome of our "We believe in moving up the stack" value that sometimes the right role for a person isn't with dbt Labs. We adopt an abundance mindset, and are confident that each of us can find the right role, whether at dbt Labs or elsewhere. This severance policy enables terminated employees the space to do just that.
+
+#### 4. The team needs information to move forwards - departure announcement guidelines for managers.
+
+❗ Managers should partner with their [People Business Partner](https://www.notion.so/dbtlabs/Find-your-PBP-f54e1de2eefa413bad7eb251a3a7c19f?pvs=4) to draft the message and communication plan.
+
+Our goal with this messaging is to manage the tension between our value of transparency, respecting the privacy of the departing employee, and creating unnecessary stress within the organization. It’s important to communicate departures immediately with those that worked closely with the individual, and we strive to do so in a way that shares the facts, addresses any concerns and allows everyone a path forward with their work.
+
+Learn more about departure message guidelines [here](https://www.notion.so/dbtlabs/Involuntary-Terminations-12bd6115f88e4874a300b272f4aec023?pvs=4#74ca657d514648edafb255bf057535f2).
+
+#### 5. Terminations are an organizational failure and we do not fail silently.
+
+We regularly conduct retrospectives on terminations. The purpose of this retrospective is to evaluate what the company, hiring team, or manager could have done differently to have prevented this outcome or to have arrived at the outcome via a better path. The purpose of this group is **not** to rehash the failures of the terminated employee.
+
+As in all things, our goal here is to be as transparent as possible. In going through these meetings we will look to share as many learnings from these retrospectives as possible without violating employee privacy.
+
+## US Employement Posters
+
+Please review the employment posters for the state you live in, and also the federal posters which apply to each US employee. All posters can be found [here](https://www.notion.so/dbtlabs/US-Employment-Posters-de219cf7a3774a73b206060831193c0c?pvs=4).
+
+## Privacy Notices
+
+Pelase review the privacy notices located [here](https://www.notion.so/dbtlabs/US-Employment-Posters-de219cf7a3774a73b206060831193c0c).
+
+## [Background Checks](https://github.com/dbt-labs/corp/blob/new-handbook-launch/handbook.md#background-checks)
```

Comment:
This is here twice, suggest removing the one with the link because I'm unsure if we should keep the link (?)